src/transformers/modeling_prophetnet.py

```diff
@@ -0,0 +1,2063 @@
+
+
+logger = logging.getLogger(__name__)
+
+_CONFIG_FOR_DOC = "ProphenetConfig"
+_TOKENIZER_FOR_DOC = "ProphetNetTokenizer"
+
+PROPHETNET_PRETRAINED_MODEL_ARCHIVE_LIST = [
+    "microsoft/prophetnet-large-uncased",
+    # See all ProphetNet models at https://huggingface.co/models?filter=prophetnet
+]
+
+
+PROPHETNET_START_DOCSTRING = r"""
+    This model inherits from :class:`~transformers.PreTrainedModel`. Check the superclass documentation for the generic
+    methods the library implements for all its model (such as downloading or saving, resizing the input embeddings,
+    pruning heads etc.)
+
+    Original ProphetNet code can be found at <https://github.com/microsoft/ProphetNet> .
+    Checkpoints were converted from original Fairseq checkpoints. For more information on
+    the checkpoint conversion, please take a look at the file ``convert_prophetnet_original_pytorch_checkpoint_to_pytorch.py``.
+
+    This model is a PyTorch `torch.nn.Module <https://pytorch.org/docs/stable/nn.html#torch.nn.Module>`_ sub-class. Use it as a regular PyTorch Module and
+    refer to the PyTorch documentation for all matters related to general usage and behavior.
+
+    Parameters:
+        config (:class:`~transformers.ProphetNetConfig`): Model configuration class with all the parameters of the model.
+            Initializing with a config file does not load the weights associated with the model, only the configuration.
+            Check out the :meth:`~transformers.PreTrainedModel.from_pretrained` method to load the model weights.
+"""
+
+PROPHETNET_GENERATION_EXAMPLE = r"""
+    ProphetNet Summarization example::
+
+        from transformers import ProphetNetTokenizer, ProphetNetForConditionalGeneration, ProphetNetConfig
+
+        model = ProphetNetForConditionalGeneration.from_pretrained('microsoft/prophetnet-large-uncased-cnndm')
+        tokenizer = ProphetNetTokenizer.from_pretrained('microsoft/prophetnet-large-uncased-cnndm')
+
+        ARTICLE_TO_SUMMARIZE = "USTC was founded in Beijing by the Chinese Academy of Sciences (CAS) in September 1958. The Director of CAS, Mr. Guo Moruo was appointed the first president of USTC. USTC's founding mission was to develop a high-level science and technology workforce, as deemed critical for development of China's economy, defense, and science and technology education. The establishment was hailed as \"A Major Event in the History of Chinese Education and Science.\" CAS has supported USTC by combining most of its institutes with the departments of the university. USTC is listed in the top 16 national key universities, becoming the youngest national key university.".lower()
+        inputs = tokenizer([ARTICLE_TO_SUMMARIZE], max_length=100, return_tensors='pt')
+
+        # Generate Summary
+        summary_ids = model.generate(inputs['input_ids'], num_beams=4, max_length=512, early_stopping=True)
+        print([tokenizer.decode(g) for g in summary_ids])
+    xProphetNet xGLUE News Title Generation example:
```

Comment:
Does this render correctly in the docs?

Reply:
Great catch - the docstring was outdated - deleted it.